pages/templates/pages/group_resource_page.html

```diff
@@ -58,10 +64,14 @@ <h3 class="panel-title">Contents</h3>
         );
       });
 
-      // Update layout and display table of contents
-      $contentBody.removeClass('col-md-offset-3');
       $contentToc.removeClass('hidden');
     }
+
+    if ($headings.length || $is_authenticated) {
+      $contentSidebar.removeClass('hidden');
+      $contentBody.addClass('col-md-9');
+      $contentBody.removeClass('col-md-8 col-md-offset-2');
```

Comment:
This logic for layout is strange to me. If there is no sidebar, then main content area is 8 columns. And if there is a sidebar, then main content is 9 columns. So that means the main content is larger area when there is also a sidebar? I would think main content area should be either the same or smaller when sidebar is added? Or in other words, when sidebar is absent, main content area should be same or larger?

Reply:
The only reason it is 8 columns is because you can't center 9 columns within bootstrap, and I didn't want to rewrite the whole layouts just for this one scenario. Without sidebar content should be centered, so had to go to 8 there.

Reply:
It's relatively minor but still seems strange to me.